src/components/about/About.scss

```diff
@@ -0,0 +1,35 @@
+.about {
+    background-color: rgba(8, 11, 209, 0.712);
+    min-height: 100vh;
+  }
+  
+  .about__wrap {
+  
+    background-color: #fff;
+  }
+  
+  .about__content {
+      display: flex;
+      width: 100%;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      padding: 5% 0 5% 0;
```

Comment:
padding with `%` looks a little bit weird, could it be done in another way?

Reply:
replaced with rem.